extension/zpagesextension/factory.go

```diff
@@ -44,7 +44,7 @@ func (f *Factory) CreateDefaultConfig() configmodels.Extension {
 			TypeVal: typeStr,
 			NameVal: typeStr,
 		},
-		Endpoint: "localhost:55679",
```

Comment:
Similar concern here, although I don't know if zpages exposes anything sensitive or not.